recipes/cudasirecon/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+c_compiler_version:        # [linux]
```

Comment:
Can you add a comment why this is needed? Probably because of CUDA but it would be nice to have that persisted here.

Reply:
yeah, it's for CUDA.  I just tried without it and got the [following error](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=335499&view=logs&j=232b7f5e-8d64-5438-fada-8478f379d226&t=e8516d7e-1153-5090-5807-53ae833fddd8&l=737):
```
#error -- unsupported GNU version! gcc versions later than 8 are not supported!
      138 | #error -- unsupported GNU version! gcc versions later than 8 are not supported!
          |  ^~~~~
```
could probably use 8 too... but it's not important for my build

Reply:
that specific config was borrowed from @jakirkham's suggestion here https://github.com/conda-forge/staged-recipes/pull/14423#issuecomment-812165914

Reply:
(comment added)

Reply:
Yeah we can probably remove this once a migration PR shows up in the feedstock, but we need it for now. In particular CUDA 10.2 is not compatible with GCC 9 (it requires GCC 7)

Reply:
still learning about migration PRs and stuff here... can you clarify what _which_ feedstock you're referring to there?  It's still a bit unclear to me whether I should be "watching a specific place" for these migrations that I can add to my ci_support folder...

Reply:
When this PR is merged, it will create a new feedstock. That's the one being referred to. Ofc this does not exist yet

Bots sometimes open PRs to a feedstock. This typically happens when there is a new version or a migration. Watching the feedstock (repo) should be sufficient to see these PRs

Anyways wouldn't worry about this for now

Reply:
thanks... yeah, I got that part from the cudadecon one.  I've noticed that some repos seem to have migrations in their ci_support folders in order to trigger multi platform builds, whereas other repos have it all encapsulated in their `build_config.yml` (which renders into all of the configs).  I'm relatively comfortable with the build config... but the migrations seemed a bit mysterious to me.

anyway, just curious.  thanks again for your help!